qiita_db/test/test_job.py

```diff
@@ -21,12 +21,36 @@ class JobTest(TestCase):
     """Tests that the job object works as expected"""
 
     def setUp(self):
-        self.job = qdb.job.Job(1)
-        self.options = {"option1": False, "option2": 25, "option3": "NEW"}
         self._delete_path = []
         self._delete_dir = []
+
+        # creating a new job for testing
+        self.options = {"option1": False, "option2": 25, "option3": "NEW"}
         _, self._job_folder = qdb.util.get_mountpoint("job")[0]
-        self.job_id = 4
+
+        self.job_create_params = ("18S", "Alpha Rarefaction",
+                                  self.options, qdb.analysis.Analysis(1),
+                                  qdb.reference.Reference(1),
+                                  qdb.software.Command(3))
+        self.job = qdb.job.Job.create(*self.job_create_params)
+        self.job_id = self.job.id
+
+        # adding filepaths
+        # file
+        self.fp = join(self._job_folder, "%d_job_result.txt" % self.job_id)
+        if not exists(self.fp):
+            with open(self.fp, 'w') as f:
```

Comment:
Just tested and they get removed once the job is deleted so they are automatically removed.